internal/future/statestore/redis.go

```diff
@@ -352,3 +344,10 @@ func (rb *redisBackend) DeindexTicket(ctx context.Context, id string, indices []
 func (rb *redisBackend) FilterTickets(context.Context, []pb.Filter) ([]string, error) {
 	return nil, status.Error(codes.Unimplemented, "not implemented")
 }
+
+func handleConnectionClose(conn *redis.Conn) {
+	err := (*conn).Close()
+	if err != nil {
+		redisLogger.Tracef("failed to close redis client connection: %s", err.Error())
```

Comment:
We tend to have `err` in `logrus.WithFields({"error": err}).Debug("")`

nit: `Debugf`